internal/pkg/config/config.go

```diff
@@ -228,22 +229,19 @@ var readConfigCached *Config
 var readConfigCachedErr error
 
 func ReadConfig() (*Config, error) {
-	loadedFromCache := true
```

Comment:
This `loadedFromCache`thing is quite old, and I've always wondered what it was for. I've thought about removing it in the past but I didn't feel it was worth it by itself.

It doesn't seem necessary, removing it looks ok to me.

Reply:
Yeah, I think this particular value was from some prior debugging.